src/vm/codeman.h

```diff
@@ -433,11 +433,11 @@ typedef struct _HeapList
     DWORD               cBlocks;        // Number of allocations
     bool                bFull;          // Heap is considered full do not use for new allocations
     bool                bFullForJumpStubs; // Heap is considered full do not use for new allocations of jump stubs
-
-#if defined(_TARGET_AMD64_)
+#if defined(_TARGET_ARM64_)
+    BYTE                padding[2];
+#endif
```

Comment:
This is the cleaner way to go as most of the common call sites require CLRPersonalityRoutine to be byte. 
Where do i put an assert for this memory alignment ?
There is an Assert in its usage in https://github.com/dotnet/coreclr/blob/master/src/vm/arm64/cgencpu.h#L199, which i hope is good enough